src/heap_jemalloc.cpp

```diff
@@ -44,7 +44,17 @@ inline bool je_edata_cmp_func (je_edata_t *a, je_edata_t *b) {
 static const char *
 heap_version(void)
 {
-	return "jemalloc";
+	static std::string version;
+
+	if (version.empty()) {
+		if (g_version == enum_je_release::JEMALLOC_5_3_0)
```

Comment:
It looks to me that the version query will happen in more than one place (line 51-55 and line 623-628).
How about store the versions in a map, then in those two places, we can simply query from the map to get the version. For example.
a map = { enum_je_release::JEMALLOC_5_3_0 => jemalloc 5.3.0, enum_je_release::JEMALLOC_5_2_1=>jemalloc 5.2.0/5.2.1, ...}

then line 50-55 becomes one single line
```c++
version = the_map[g_version]
```

line  623-628 also become simple
```c++
CA_PRINT(the_map[g_version])
```